lib/gutentag/tag_names.rb

```diff
@@ -1,21 +0,0 @@
-class Gutentag::TagNames
-  def self.call(*names)
-    new(names).call
-  end
-
-  def initialize(names)
-    @names = names.flatten
-  end
-
-  def call
-    name_values.collect { |name| Gutentag::TagName.call name }
```

Comment:
Can we keep TagNames, but have it use `Gutentag.normaliser` here? (Which it really should have been doing originally.)